utils/api.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+)
+
+// API Constants
+const (
+	APIUsers = "user"
```

Comment:
This should have version.
/api/v1/user, 
because your implementation is specific for api version. This should be called as `EndPoint`?

Reply:
I'm sorry, but that's not a valid Endpoint.
Endpoint is `{{api_base_url}}/user`

That's how it is made.